### DISCUSSION TITLE
可以增加对仓库的自定义的简单描述吗

### DISCUSSION THREAD

**Discussion by miranahf** [riskers/github-plus-extension#18] [Ideas]
有些仓库名并不是语义化的，有可能 star 的时候知道是干啥的，但是过一段时间就忘了，还得重新读文档，star 列表多了就很头疼。标签并不能很好的满足这个需求，因此希望增加一段可自定义描述文字的功能

**Comment by anonymouswhac** [comment]
+1 这个需求相信很多朋友都有

**Comment by riskers** [comment]
感谢使用以及提出建议，最近我可能会重构一下 GithubX，这个需求不难。
